core/corerepo/gc_test.go

```diff
@@ -0,0 +1,186 @@
+package corerepo
+
+import (
+	"io"
+	"testing"
+
+	context "github.com/ipfs/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
+
+	bstore "github.com/ipfs/go-ipfs/blocks/blockstore"
+	key "github.com/ipfs/go-ipfs/blocks/key"
+	"github.com/ipfs/go-ipfs/core"
+	importer "github.com/ipfs/go-ipfs/importer"
+	chunk "github.com/ipfs/go-ipfs/importer/chunk"
+	dag "github.com/ipfs/go-ipfs/merkledag"
+	pin "github.com/ipfs/go-ipfs/pin"
+	u "github.com/ipfs/go-ipfs/util"
+)
+
+func TestBasicSeparation(t *testing.T) {
```

Comment:
some blocks should be internal only, like for example keys. could add the private key or something. 

sure that's fine.